docs/guide.md

```diff
@@ -770,18 +770,18 @@ by selecting the option `None` (this is the default option).
 
 ## Automation Tools
 
-An automation tool is software or a platform designed to automate repetitive 
+An automation tool is software or a platform designed to automate repetitive
 tasks, processes, or workflows that are traditionally performed manually.
-These tools are used to streamline and optimize various operations, reduce 
+These tools are used to streamline and optimize various operations, reduce
 human intervention, increase efficiency, and minimize errors.
 Currently SciCookie is using `make` and `Makim`
```

Comment:
Currently, SciCookie allows you to use `make`  and/or `Makim`.

Reply:
thanks for your suggeston @Anavelyz 

```suggestion
Currently, SciCookie allows you to use `Make` and/or `Makim`.
```